banyan-cli/src/tag_index.rs

```diff
@@ -101,6 +101,7 @@ pub fn map_to_index_set(table: &TagSet, tags: &TagSet) -> Option<IndexSet> {
         .collect::<Option<_>>()
 }
 
+#[allow(dead_code)]
```

Comment:
are there plans to use actyx-sdk in banyan?

Reply:
Dunno yet. Let's see how it evolves.